translations/es.ts

```diff
@@ -1049,7 +1049,7 @@ Disabling chat history now will leave the encrypted history intact (but not usab
         <location filename="../src/widget/form/settings/generalsettings.ui" line="153"/>
         <source>Enable light tray icon.</source>
         <comment>toolTip for light icon setting</comment>
-        <translation>Habilitar ícono claro en la bandeja.</translation>
+        <translation>Habilitar ícono claro en la bandeja</translation>
```

Comment:
Also in English. Sentences end correctly with a period (full stop), question mark or exclamation mark. The original string you can read as an imperative sentence. But it is more natural to read it as an inifinitive - in that case it is not a sentence.